### PR TITLE
SRFCMSDA-3507-swipeable-unswipeable

### PR DESCRIPTION
**Pull request by pchr-srf**

### CODE REVIEW

File: source/_patterns/30-organisms/collection/collection.scss [hunk 1]
```diff
@@ -519,6 +519,36 @@ $background-overlap-wide: 50px;
       }
     }
   }
+
+  // only one teaser on mobile: remove 40px gap on the right
+  @include smartphone {
+    .collection__teaser-item:last-child:first-child {
```
Comment (albuvee) [inline source/_patterns/30-organisms/collection/collection.scss]: Sexy! 😎

File: source/_patterns/30-organisms/collection/collection.scss [hunk 2]
```diff
@@ -519,6 +519,36 @@ $background-overlap-wide: 50px;
...
+    .collection__teaser-item:nth-last-child(-n+3):first-child ~ .collection__teaser-item {
+      flex-basis: calc((100% - 40px) / 3.00001);
+      padding-right: 24px;
+      max-width: none;
+
+      @include desktop-wide {
+        padding-right: 32px;
+      }
+
+        &:last-child{
```
Comment (albuvee) [inline source/_patterns/30-organisms/collection/collection.scss]: indentation (and missing space «…ild{»)

File: source/_patterns/30-organisms/collection/collection.scss [hunk 3]
```diff
@@ -519,6 +519,36 @@ $background-overlap-wide: 50px;
...
+        min-width: 0;
+        margin-right: 8px;
+
+          &::after{
```
Comment (albuvee) [inline source/_patterns/30-organisms/collection/collection.scss]: ditto missing space

File: source/_patterns/30-organisms/collection/collection.scss (hunk 2 above)
Comment (pchr-srf) [reply source/_patterns/30-organisms/collection/collection.scss]: hesch recht! Gefixt

File: source/_patterns/30-organisms/collection/collection.scss (hunk 3 above)
Comment (pchr-srf) [reply source/_patterns/30-organisms/collection/collection.scss]: dito gefixt

**Pull request merged**